script/helpers/verifyConstants.ts

```diff
@@ -32,16 +32,33 @@ export const verifyConstants = async () => {
       }
       let { superTokenInfo } = currentPc;
       checkMissingFields({ superTokenInfo });
-      let { name, symbol, decimals, initialSupplyOwner, owner, initialSupply } =
-        superTokenInfo!;
-      checkMissingFields({
-        name,
-        symbol,
-        decimals,
-        initialSupplyOwner,
-        owner,
-        initialSupply,
-      });
+
+      if ("address" in superTokenInfo) {
```

Comment:
are we planning to keep address in superTokenInfo object? i believe would be better to keep all the addresses at one place in existing-token-addresses file.

Reply:
The idea was here that if we supply a token, then we won't need to supply any other information. Hence the token already exists and we don't need to deploy anymore